### DISCUSSION TITLE
How to pass 'bitrate' option to exhale encoder via freaccmd?

### DISCUSSION THREAD

**Discussion by santiago046** [enzo1982/freac#409] [Help & Support] [answered]
Hi, I am using `freaccmd` and I am having trouble to pass the exhale's bitrate parameter.

I tried like this:
```
>>> freaccmd -e exhale -- 4 ~/music/*.flac
```
But it treats ‘4’ as an input file.

**Comment by santiago046** [comment]
Do I need to provide more information to get help?

I don't want to bother, but I would really like to know if I'm doing something wrong and so I can solve it.

**Comment by enzo1982** [comment, marked as answer]
Hi, I can confirm the issue. It is currently not possible to use the exhale encoder with freaccmd due to this.

I'm investigating possible options to fix this with the goal to include a fix in fre:ac 1.1.7 (which should be ready around the end of the month).
